compose.yml

```
@@ -23,6 +23,7 @@ services:
       AAS_REPO_API_URL: 'http://mnestix-api:5064/repo'
       SUBMODEL_REPO_API_URL: 'http://mnestix-api:5064/repo'
       MNESTIX_BACKEND_API_URL: 'http://mnestix-api:5064'
+      MNESTIX_BACKEND_API_KEY: 'verySecureApiKey'
```

we use it in the backend already, so maybe
```suggestion
      MNESTIX_BACKEND_API_KEY: ${MNESTIX_BACKEND_API_KEY:-verySecureApiKey}
```

---

but then it will not work... i've tried it